governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx

```diff
@@ -804,7 +826,7 @@ const General = ({ proposerServerUrl }: { proposerServerUrl: string }) => {
     if (connected && squads) {
       const provider = new AnchorProvider(
         connection,
-        squads.wallet,
+        squads.wallet as Wallet,
```

Comment:
upgraded `@coral-xyz/anchor` version and now `AnchorProvider` requires a slightly different `Wallet` interface but `squads.wallet` is still using an old version of `@projectserum/anchor` package